src/libs/SuggestionUtils.ts

```diff
@@ -1,22 +1,11 @@
 import CONST from '@src/CONST';
 
-/**
- * Trims first character of the string if it is a space
- * @param {String} str
- * @returns {String}
- */
-function trimLeadingSpace(str) {
-    return str.slice(0, 1) === ' ' ? str.slice(1) : str;
+/** Trims first character of the string if it is a space */
```

Comment:
To keep comment styles consistent with the rest of our code, for method docs, we always prefer multiple lines.
```suggestion
/** 
 * Trims first character of the string if it is a space 
 */
```

(same for below)

Reply:
@tgolen done!